src/include/mutex.h

```diff
@@ -58,6 +58,15 @@ struct __wt_rwlock { /* Read/write lock */
 
     WT_CONDVAR *cond_readers; /* Blocking readers */
     WT_CONDVAR *cond_writers; /* Blocking writers */
+
+#ifdef TSAN_BUILD
+    /*
+     * Our read/write locks provide thread safety but TSan reports false data races for fields
+     * protected by these locks. To help TSan we perform dummy atomic reads and writes to this field
```

Comment:
I would clarify the reason TSan reports those races, which is that it doesn't support our barrier implementations which our rw_lock implementation depends on.

Reply:
Updated in [fccde73](https://github.com/wiredtiger/wiredtiger/pull/10751/commits/fccde7399554b76646298dcbccfe11a76a8e4555), along with a fixed typo